### DISCUSSION TITLE
Any new release on the horizon?

### DISCUSSION THREAD

**Discussion by IVazharov** [FlareSolverr/FlareSolverr#1350] [Q&A]
Hello,

Since the 3.3.21 release currently is not functional in Windows can we expect a new fixed release any time soon?
I wouldn't ask this question if I could find a way to build the Windows x64 release based on a PR myself.

Thanks in advance if anyone is prepared to answer this!

**Comment by ilike2burnthing** [comment]
To build from source - https://github.com/FlareSolverr/FlareSolverr/pull/1163#issuecomment-2067972723
